src/main/steps/applicant1/confirm-your-joint-application/template.njk

```diff
@@ -70,10 +70,6 @@
     <h3 class="govuk-heading-m">{{ subHeading10 }}</h3>
     <p class="govuk-body">{{ line20 }}</p>
   {% endif %}
-  <h3 class="govuk-heading-m">{{ subHeading11 }}</h3>
-  <p class="govuk-body">{{ confirmApplicationIsTrue | safe }}</p>
```

Comment:
You've got rid of confirmApplicationIsTrue here but not in the content is that correct?

Reply:
its used somewhere else